src/simulators/tensor_network/tensor_network_state.hpp

```diff
@@ -336,16 +345,16 @@ void State::initialize_omp() {
     qreg_.set_omp_threads(BaseState::threads_); // set allowed OMP threads in qubitvector
 }
 
-  void State::initialize_creg(uint_t num_memory, uint_t num_registers) {
-    cout << "initialize_creg not supported yet" <<endl;
-  }
+void State::initialize_creg(uint_t num_memory, uint_t num_registers) {
+  cout << "initialize_creg not supported yet" <<endl;
```

Comment:
Enclose this with `#ifdef DEBUG` or just leave a comment and lave this method empty.
If enclosing, don't need to append `<<endl` in the end, just write:
```c++
#ifdef DEBUG
cout << "tensor_network_state::initialize_creg() not supported yet\n";
#endif
```

Reply:
enclosed in #ifdef DEBUG